net/netcheck/netcheck.go

```diff
@@ -63,6 +63,9 @@ const (
 	// icmpProbeTimeout is the maximum amount of time netcheck will spend
 	// probing with ICMP packets.
 	icmpProbeTimeout = 1 * time.Second
+	// httpsProbeTimeout is the maximum amount of time netcheck will spend
+	// probing over HTTPS.
```

Comment:
seems short? with TCP + TLS round trips, that's not many RTTs in a second. Do we only care about close things? Say so in the comment if so?

Reply:
I was looking at a measurement dataset that was post-TCP/TLS handshakes when I originally set this, not realizing it excluded them. Updated in https://github.com/tailscale/tailscale/pull/13395/commits/1467db765b9e0bc1be779581a22b172e7ec9cbe3 to revert to 5s/consume whatever time is left, but with more docs.